package.json

```diff
@@ -7,6 +7,7 @@
     "server1": "node src/server.js",
     "server2": "python src/mastodonApi/mastodonApi.py",
     "test": "jest --maxWorkers==1",
+    "mytest": "jest --maxWorkers==1 recommend.test.js",
```

Comment:
i think you dont need to define a separate "mytest" var - it will run automatically when we call `npm test`

Reply:
I just write it for test...